### ISSUE TITLE
ggsave error with emf and wmf "grDevices::win.metafile(...) : unused argument (bg = "white")"

### ISSUE THREAD

**Issue by dweis-ku** [tidyverse/ggplot2#4521]
ggsave produces error when saving in wmf or emf format
Error in grDevices::win.metafile(...) : unused argument (bg = "white")

Save as PDF and PNG work as expected.

reprex:
```r
library(tidyverse)
e <- ggplot(mpg, aes(cty, hwy))
e + geom_point()
ggsave("test.wmf")
ggsave("test.emf")
```
Expected that my plot would save.

**Comment by dweis-ku** [comment]
Further troubleshooting:
 The following code generates a wmf file successfully:
````r
library(tidyverse)
e <- ggplot(mpg, aes(cty, hwy))
e <- e + geom_point()
win.metafile("test.wmf")
print(e)
dev.off()
````

**Comment by yutannihilation** [comment]
I don't have Windows machine right now, but it seems this is caused by the change of #4164.

**Comment by dweis-ku** [comment]
resolved in ggplot 3.3.5

**Comment by yutannihilation** [comment]
Thanks for confirming!
